tests/functions/test_alias.py

```diff
@@ -43,6 +45,114 @@ def test_alias_equal():
 
     req = OriginRequest(conf_file=TEST_CONF)
 
-    aliases = [{"src": "/foo/bar", "dest": "/quux"}]
+    aliases = [{"src": "/foo/bar", "dest": "/quux", "exclude_paths": []}]
```

Comment:
Please be sure to test alias resolution for aliases without an `exclude_paths` key.